views/Demo.jsx

```diff
@@ -55,7 +56,12 @@ export default React.createClass({
     } = this.state;
 
     return (
-      <div className="_container _container_large">
+      <div className="demo _container _container_large">
```

Comment:
Can we add a link to `cloud.ibm.com` instead of `console.bluemix.net` for floating CTA?